core/lib/workarea/configuration.rb

```diff
@@ -1306,6 +1306,12 @@ def self.setup_defaults
         cvv
         amount
       ]
+
+      # Tax code assigned to shipping options when an existing service does
+      # not exist. This is useful for third-party gateways to assign tax codes
+      # to dynamically generated options. This can either be a string or a proc.
+      # If using a proc, the carrier and name of the service will be passed.
+      config.default_shipping_service_tax_code = nil
```

Comment:
Any reason we shouldn't make this admin facing?

Reply:
admin can't define procs?

Reply:
the code looks like it would support both!

Reply:
Not sure what you're saying